### ISSUE TITLE
ktlint issue on library usage

### ISSUE THREAD

**Issue by praveentcom** [mapbox/mapbox-maps-flutter#22]
`flutter run` gives an error related to referencing ktlint.gradle.

```
* Where:
Build file '/Users/praveen/development/flutter/.pub-cache/hosted/pub.dartlang.org/mapbox_maps_flutter-0.3.0/android/build.gradle' line: 57

* What went wrong:
A problem occurred evaluating project ':mapbox_maps_flutter'.
> Could not read script '/Users/praveen/***/android/gradle/ktlint.gradle' as it does not exist.
```

Upon checking the library, it's referencing these two files on the root directory.

```
project.apply {
    from("$rootDir/gradle/ktlint.gradle")
    from("$rootDir/gradle/lint.gradle")
}
```

The end user need not have these files.

**Comment by dawid-niedzwiecki** [comment]
Same problem here

**Comment by praveentcom** [comment]
Temporarily unblocked by copy-pasting the two files to my project directory.

Expecting the library to be patched.

**Comment by yunikkk** [comment]
This is already patched in main and will be fixed in the next minor release.

**Comment by geisterfurz007** [comment]
For those not looking to copy things around or moving to main as ref, you can just create `ktlint.gradle` and `lint.gradle` as empty files.

**Comment by yunikkk** [comment]
Should be fixed with 0.4.0.
